VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/select-hearing-layout/select-hearing-layout.component.spec.ts

```diff
@@ -13,13 +14,20 @@ describe('SelectHearingLayoutComponent', () => {
     const translateService = translateServiceSpy;
     const headingButton = document.createElement('button');
     const textButton = document.createElement('button');
+    const buttonContentKeyWhenOpen = 'open-all';
+    const buttonContentKeyWhenClosed = 'close-all';
 
     beforeEach(() => {
         conference = new ConferenceTestData().getConferenceDetailNow();
         component = new SelectHearingLayoutComponent(videoCallService, translateService);
         component.conference = conference;
         textButton.innerHTML = 'Open all';
         document.getElementById = jasmine.createSpy('accordion-choose-layout-heading').and.returnValue(headingButton);
+        document.getElementsByClassName = jasmine.createSpy('govuk-accordion__section-button').and.returnValue({
+            item() {
+                return headingButton;
+            }
+        });
```

Comment:
I'm not really sure what's going on here, even with the old code. Do lines 26-30 not get overwritten by 31-35 or can they co-exist because of the different names?

Reply:
strange, I thought this is the way of mocking the dom document with expected argument. but I am assuming in jasmine the mocks should have been something like:
`spyOn(document, "getElementsByClassName ").withArgs(`govuk-accordion__section-button`).and.returnValue("whatever")`
Anyways, i have removed the line I have added its not required I think I had it from earlier so once again good spot! :D